airflow/www/security.py

```diff
@@ -360,8 +365,9 @@ def merge_pv(perm, view_menu):
             .filter(or_(models.DagModel.is_active, models.DagModel.is_paused))\
             .filter(~models.DagModel.is_subdag).all()
 
+        # create can_dag_edit and can_dag_read permissions for every dag(vm)
         for dag in all_dags_models:
-            for perm in dag_perms:
+            for perm in DAG_PERMS:
                 merge_pv(perm, dag.dag_id)
```

Comment:
Understand the idea here is each DAG will be considered as a `View`.

One rare (but potentially valid) case to consider: what if a DAG id is identical to an existing View name?

Reply:
It shouldn't have any impact. For instance, assuming the dag name is test-dag, this will add two entries to the table: can-dag-edit with test-dag and can-dag-read with test-dag.